setup.cfg

```diff
@@ -96,21 +96,6 @@ skip=
     pandas/util/_print_versions.py,
     pandas/util/_decorators.py,
     pandas/util/_test_decorators.py,
-    pandas/io/s3.py,
-    pandas/io/parquet.py,
-    pandas/io/feather_format.py,
-    pandas/io/api.py,
-    pandas/io/sql.py,
-    pandas/io/clipboards.py,
-    pandas/io/excel.py,
-    pandas/io/date_converters.py,
-    pandas/io/testing.py,
```

Comment:
I guess as an extra callout this isn't in the diff, though I also don't see it existing as a module

Reply:
Good spot - my apologies I don't think this file ever existed in pandas running `git log --full-history  -- *pandas/io/testing.py` confirms this. I think I had a local file named this at one stage and it must have been present when I created the list of files to ignore.